packages/yarnpkg-core/sources/structUtils.ts

```diff
@@ -460,37 +659,76 @@ function prettyRangeNoColors(range: string): string {
   }
 }
 
+/**
+ * Returns a `string` that is suitable to be printed to `stdout` (contains colors)
```

Comment:
> contains colors

Doesn't that depend if you have it enabled or not?

Reply:
It does. Rephrasing suggestions appreciated :)